testtools/_deferredmatchers.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) testtools developers. See LICENSE for details.
+
+"""Matchers that operate on synchronous Deferreds.
+
+A "synchronous" Deferred is one that does not need the reactor or any other
+asynchronous process in order to fire.
+
+Normal application code can't know when a Deferred is going to fire, because
+that is generally left up to the reactor. Well-written unit tests provide fake
```

Comment:
comment only: I realise that this is a true statement, but it sounds a little judgy.